<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlotChartPropertiesWidget</class>
 <widget class="qMRMLWidget" name="qMRMLPlotChartPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="plotDataSeriesLabel">
     <property name="text">
      <string>Plot data series:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLCheckableNodeComboBox" name="plotSeriesComboBox">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip">
      <string>Add/Remove plots data series to/from the current chart.</string>
     </property>
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLPlotSeriesNode</string>
      </stringlist>
     </property>
     <property name="noneEnabled">
      <bool>false</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
     <property name="editEnabled">
      <bool>false</bool>
     </property>
     <property name="renameEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="chartTitleLabel">
     <property name="text">
      <string>Chart title:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="titleLineEdit">
     <property name="toolTip">
      <string>Enter a title for the chart.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="xAxisTitleLabel">
     <property name="text">
      <string>X axis title:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="xAxisLabelLineEdit">
     <property name="toolTip">
      <string>Enter a label for the X-axis.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="xAxisRangeLabel">
     <property name="text">
      <string>X axis range:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="xAxisManualRangeCheckBox">
       <property name="toolTip">
        <string>Enable manual setting of X axis range for all views of this chart</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="xAxisRangeMinDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="xAxisRangeMaxDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="yAxisRangeLabel">
     <property name="text">
      <string>Y axis range:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="yAxisManualRangeCheckBox">
       <property name="toolTip">
        <string>Enable manual setting of Y axis range for all views of this chart</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="yAxisRangeMinDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSpinBox" name="yAxisRangeMaxDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="yAxisTitleLabel">
     <property name="text">
      <string>Y axis title:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="yAxisLabelLineEdit">
     <property name="toolTip">
      <string>Enter a label for the Y-axis.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="fontTypeComboBox">
     <item>
      <property name="text">
       <string>Arial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Times</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Courier</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="fontTypeLabel">
     <property name="text">
      <string>Font Type:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="titleFontSizeLabel">
     <property name="text">
      <string>Chart title font size:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="ctkDoubleSpinBox" name="titleFontSizeDoubleSpinBox">
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="axisTitleFontSizeLabel">
     <property name="text">
      <string>Axis title font size:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="ctkDoubleSpinBox" name="axisTitleFontSizeDoubleSpinBox">
     <property name="value">
      <double>16.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="axisLabelFontSizeLabel">
     <property name="text">
      <string>Axis label font size:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="ctkDoubleSpinBox" name="axisLabelFontSizeDoubleSpinBox">
     <property name="value">
      <double>12.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="legendVisibleLabel">
     <property name="text">
      <string>Legend visibility:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="legendVisibleCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="gridVisibleLabel">
     <property name="text">
      <string>Grid visibility:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="legendFontSizeLabel">
     <property name="text">
      <string>Legend font size:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="ctkDoubleSpinBox" name="legendFontSizeDoubleSpinBox">
     <property name="value">
      <double>16.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="logScaleLabel">
     <property name="text">
      <string>Logarithmic scale:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="xAxisLogScaleCheckBox">
       <property name="text">
        <string>X axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="yAxisLogScaleCheckBox">
       <property name="text">
        <string>Y axis</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="gridVisibleCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>plotSeriesComboBox</tabstop>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>xAxisLabelLineEdit</tabstop>
  <tabstop>xAxisManualRangeCheckBox</tabstop>
  <tabstop>yAxisManualRangeCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>xAxisManualRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xAxisRangeMinDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xAxisManualRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>xAxisRangeMaxDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisManualRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxisRangeMinDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yAxisManualRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxisRangeMaxDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
